aten/src/ATen/mps/MPSDevice.mm

```diff
@@ -90,8 +90,17 @@ static inline MTLLanguageVersion getMetalLanguageVersion(const id<MTLDevice>& de
 
 }
 
-bool MPSDevice::isMacOS13Plus() const {
-  return _macos13plus;
+bool MPSDevice::isMacOS13Plus(int32_t subVersion) const {
+  id mpsCD = NSClassFromString(@"MPSGraph");
+  static bool _macos_13_0_plus = [mpsCD instancesRespondToSelector:@selector(cumulativeSumWithTensor:axis:name:)] == YES;
+  static bool _macos_13_1_plus = [mpsCD instancesRespondToSelector:@selector(
+    sampleGridWithSourceTensor:coordinateTensor:layout:normalizeCoordinates:relativeCoordinates:alignCorners:paddingMode:samplingMode:constantValue:name:)] == YES;
+
+  switch (subVersion) {
+    case 0:  return _macos_13_0_plus;
```

Comment:
Can we replace these raw numbers 0, 1, etc., with enums like `MACOS_VER_13_0_PLUS`, `MACOS_VER_13_1_PLUS`, and etc.?

Reply:
Or you'll do that in a separate PR?

Reply:
Lets add in this PR.

Reply:
Thanks. Updated